src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/Authentication/RemoteAppAuthenticationService.cs

```diff
@@ -51,7 +54,7 @@ public Task InitializeAsync(AuthenticationScheme scheme)
         // Finish initializing the http client here since the scheme won't be known
         // until the owning authentication handler is initialized.
         _options = _authOptionsSnapshot.Get(scheme.Name);
-        _client.BaseAddress = new Uri(_remoteAppOptions.RemoteAppUrl, _options.AuthenticationEndpointPath);
+        _client.BaseAddress = new Uri($"{_remoteAppOptions.RemoteAppUrl.ToString().TrimEnd('/')}{_options.AuthenticationEndpointPath}");
```

Comment:
should we stash this as an internal property on the options so we're not allocaitng each request with `.ToString().TrimEnd('/')`? I'm assuming this InitializeAsync is an every request thing

Reply:
I think it's only called once per auth handler (this method is called from the authenticaiton handler's `InitializeHandlerAsync`) but, even so, I'm fine with storing this in a field in case anything changes in the future.

Reply:
Actually, looking more at this, it's hard to persist this anywhere because the path depends on the options retrieved in the initialize method. So, if it were called multiple times, it might have different values each time.

I'm going to leave this as-is but will test to make sure it's not called multiple times.